src/lib/utils/functions/sqlFunctions.ts

```diff
@@ -78,10 +101,7 @@ export const insertConditionsFromSearchParams = (
   const tokens: string[][] = lexer.tokenize(sql)
   // we want to either create a where clause or insert into the where clause
   const foundIndex = tokens.findIndex(el => el[0] === 'WHERE')
-  const whereClauseIndex =
-    foundIndex === -1
-      ? tokens.findIndex(el => el[0] === 'FROM') + 2
```

Comment:
This is one bug.  We can't assume that we should insert the WHERE token 2 tokens after the FROM token (the DOT version notation messes this up).

Reply:
This is interesting. Good catch!